### ISSUE TITLE
[1.18.1] [BUG] Bee info text always reads 0 regardless off number of bees in hive 

### ISSUE THREAD

**Issue by joylessdave** [maruohon/minihud#201]
running minecraft 1.18.1 
minihud version minihud-fabric-1.18.0-0.19.0-dev.20211204.230657.jar

whenever you look at a bee hive the infotext on teh overlay always reads 0 even when you watch a bee enter the hive

**Comment by maruohon** [comment]
Like the tooltip says, this only works in single player. The vanilla server does not sync the full block entity NBT data for everything to the clients. Support for requesting and syncing it will be in my Servux mod at some point in the future.

You can get it working currently if you add the PCA Carpet extension mod to the server, and the MasaGadget mod to the client. This would obviously require the server to be running Fabric.
https://github.com/plusls/MasaGadget/releases
https://github.com/plusls/plusls-carpet-addition/releases

**Comment by joylessdave** [comment]
oh didnt see that in the tooltip, it was working in 1.17.1 as i was using it in november

**Comment by maruohon** [comment]
Was that in single player then? It never worked in multiplayer unless the additional mods mentioned in the last message are present. Well technically it's possible that the server would sync the full NBT data to the clients all the time, but AFAIK no server variant does that, as it would make no sense and would be bad for network bandwidth and server performance.

**Comment by joylessdave** [comment]
no multiplayer
its a 1.17.1 fabric server I shut it down when I started a 1.18.1 world but I can fire it back up and check, but I was defiantly using it making bee farms

**Comment by maruohon** [comment]
And you mean the bee info line feature? And not the separate bee tooltip feature on the items when you hover them in the inventory?

**Comment by joylessdave** [comment]
yeah the info line as i was checking the hives has 5 in before silk touching them

**Comment by TnTBass** [comment]
Any notice when this would be in the Servux mod? I would really love to see bee information on my Fabricmc server. I'm not super keen on adding a couple other mods just to get this data.

I appreciate all you do, I love this mod.

**Comment by maruohon** [comment]
@TnTBass Still no idea when I'll have the time (and motivation) to work on all the missing things that would lead to this being available via my mods directly (because it will only come via the main development code from 1.12.2, which is still missing a number of things from all the different client mods before I'll port it to the later MC versions).

As I mentioned in my first reply, it should work currently (I think?) if you add PCA, or the separate smaller [pca-protocol](https://github.com/Fallen-Breath/pca-protocol/releases) mod (that just includes this communication protocol part of PCA) to the server, and MasaGadget to the client, and then enable the `pcaSyncProtocol` option in MasaGadget.
